src/pymachine/lexicon.py

```diff
@@ -551,7 +551,9 @@ def test_static_graph_building():
         pass
 
     def get_machine(self, printname, second=False):
+        print "PYMACHINE"
```

Comment:
Please don't leave these in the code this way. A good solution is to use logging.debug to print such messages and change loglevel to DEBUG for debugging, then changing it back before commiting